lib/active_merchant/billing/gateways/orbital/orbital_soft_descriptors.rb

```diff
@@ -27,16 +27,14 @@ def initialize(options = {})
       def validate
         errors = []
 
-        errors << [:merchant_name, "is required"] if self.merchant_name.blank?
-        errors << [:merchant_name, "is required to be 25 bytes or less"] if self.merchant_name.bytesize > 25
+        errors << [:merchant_name, 'is required'] if self.merchant_name.blank?
+        errors << [:merchant_name, 'is required to be 25 bytes or less'] if self.merchant_name.bytesize > 25
 
-        if(!empty?(self.merchant_phone) && !self.merchant_phone.match(PHONE_FORMAT_1) && !self.merchant_phone.match(PHONE_FORMAT_2))
-          errors << [:merchant_phone, "is required to follow \"NNN-NNN-NNNN\" or \"NNN-AAAAAAA\" format"]
-        end
+        errors << [:merchant_phone, 'is required to follow "NNN-NNN-NNNN" or "NNN-AAAAAAA" format'] if !empty?(self.merchant_phone) && !self.merchant_phone.match(PHONE_FORMAT_1) && !self.merchant_phone.match(PHONE_FORMAT_2)
 
-        [:merchant_email, :merchant_url].each do |attr|
+        %i[merchant_email merchant_url].each do |attr|
```

Comment:
sometimes I wonder, is it really that much of a time saver 😄  Not a comment to you, just a general comment.